### ISSUE TITLE
Why set the initial_lr as 5e-6?

### ISSUE THREAD

**Issue by limaodaxia** [zwx8981/LIQE#19]
Hello, @zwx8981 
Why set the initial_lr as 5e-6? Is it a little higher and will cause the  waving of loss, so you use cosine anneal?

Best wishes

**Comment by zwx8981** [comment]
Hi @limaodaxia , the initial_lr is set according to the performance on the validation set. Empirically, a high learning rate will result in failure of training (significant performance drop). Cosine anneal is a common good practice in training modern DNNs, so we use it. You may also try a slightly higher learning rate like 1e-5, best.
